recipes/pyamg/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# On OSX, we need to ensure we're using conda's gcc/g++
```

Comment:
Carried over from another recipe – I think it was an OpenMP thing that doesn't actually apply here. I'll remove it.